src/ECS/ECS.cpp

```diff
@@ -0,0 +1,179 @@
+#include <entt/entt.hpp>
+#include <SDL.h>
+
+#include "GameManager.h"
+
+#include "Rendering/Renderer.h"
+#include "Rendering/Texture.h"
+#include "Rendering/Camera.h"
+
+#include "Parsers/Gm1File.h"
+
+#include "System/Logger.h"
+#include "ECS.h"
+
+#include "Assets/AssetsStorage.h"
+
+using namespace Sourcehold::Rendering;
+using namespace Sourcehold::System;
+using namespace Sourcehold::Parsers;
+using namespace Sourcehold::Game;
+
+// Test of EnTT, copypasted from EnTT docs
+// It is just to check if pipeline works with new import
+
+namespace Sourcehold {
+    namespace ECS {
+        struct PositionComponent {
```

Comment:
I think it's sufficient to just call this struct Position.
You may put this in a seperate file under a namespace, if you are concerned about conflicts.

Reply:
I will exclude Components to own namespace

Reply:
I will exclude Components to own namespace

Reply:
Done